# PARAMETERS FOR MOTION TASK SERVER
# name of action server to send requests to
action_server_name: 'motion_task_server'
# node update rate, in Hz
update_rate: 25
# whether or not to use safety bond node
safety_enabled: False 
# startup timeout, in seconds
startup_timeout: 20
# force a task to cancel when requested
force_cancel: false
